propagators/ot/ot_propagator_test.go

```diff
@@ -133,3 +136,26 @@ func TestOT_Extract(t *testing.T) {
 		assert.Equal(t, trace.NewSpanContext(test.expected), sc, info...)
 	}
 }
+
+func TestOT_Fields(t *testing.T) {
+	propagator := OT{}
+
+	want := []string{traceIDHeader, spanIDHeader, sampledHeader}
+
+	got := propagator.Fields()
+
+	assert.Equal(t, want, got, "Fields()\ngot:  %v\nwant: %v", got, want)
```

Comment:
`assert` will format `got` and `want` in the message it logs. No need to duplicate.

```suggestion
	assert.Equal(t, want, got,)
```

Reply:
Good point

![image](https://github.com/user-attachments/assets/34d6e088-53f5-49f8-a902-c5cf1632ef32)